packages/select/src/Select/BaseGroupHeading.tsx

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2022-present, NDLA.
```

Comment:
Kjem til å ta litt tid å bli vant til!
```suggestion
 * Copyright (c) 2023-present, NDLA.
```